build/arch.mips.mk

```diff
@@ -8,7 +8,9 @@ ELFTYPE := elf32-littlemips
 ifeq ($(KASAN), 1)
 ifeq ($(KERNEL), 1)
 	# Added to files that are sanitized
-	CFLAGS_KASAN = -fsanitize=kernel-address --param asan-globals=1 --param asan-stack=1 -fasan-shadow-offset=0xD8000000
+	CFLAGS_KASAN = -fsanitize=kernel-address -fasan-shadow-offset=0xD8000000 \
+								 --param asan-globals=1 \
```

Comment:
Could use better alignment.

```
	CFLAGS_KASAN = -fsanitize=kernel-address -fasan-shadow-offset=0xD8000000 \
                       --param asan-globals=1 --param asan-stack=1 
```

Reply:
As GitHub displays tab using 6 spaces, the alignment will never look good here if we indent `CFLAGS_KASAN` using tab. Right now my code change looks properly in an editor that displays tab as 2 spaces.

I've discovered that the whole block within `ifeq ... endif` can be indented with spaces -- no tab before `CFLAGS_KASAN` needed. Maybe that would be the solution?

Reply:
> I've discovered that the whole block [...]
Yes, please do so.

Reply:
Fixed in https://github.com/cahirwpz/mimiker/pull/693/commits/7bc832c02d22ee09c29473c71830ed225b61ab16